packages/analytics-js-integrations/src/integrations/RedditPixel/utils.js

```diff
@@ -0,0 +1,15 @@
+import { removeUndefinedAndNullAndEmptyValues } from '../../utils/commonUtils';
+const createUserIdentifier = (traits = {}, context = {}) => {
+  const userIdentifier = {};
+  const { email, externalId, idfa, aaid } = traits;
+  userIdentifier.email = email;
+  userIdentifier.externalId = externalId;
+  userIdentifier.idfa = idfa || context.device?.idfa;
+  userIdentifier.aaid = aaid;
```

Comment:
I think both should be mapped to `context.device.advertisingId`. Acc. to the doc for reddit pixel
https://business.reddithelp.com/helpcenter/s/article/Reddit-Pixel-Advanced-Matching
```
idfa (Apple’s Identifier for Advertisers) 
aaid (Google Advertising ID)
```

Reply:
RudderStack support advertisingId in context.device
https://www.rudderstack.com/docs/event-spec/standard-events/common-fields/